CONstitution.xml

```diff
@@ -107,99 +107,100 @@
+      <text id="👽">All <toa ref="constituent">CONstituents</toa> have the inalienable right to <toa ref="motion-propose">call motions</toa> if they have enough <toa ref="fungible-pc">pc</toa> to do so and have not exceeded their daily limit.</text>
+      <text id="❔">Each CONstituent is limited to <toa ref="motion-propose">proposing</toa> 10 <toa ref="motion">motions</toa> (no matter which kind) per UTC day.</text>
+      <text id="👶">To call a <toa ref="motion-simple">simple motion</toa>, a <toa ref="constituent">CONstituent</toa> must have enough <toa ref="fungible-pc">PC</toa> to <toa ref="motion-vote">vote</toa> for that <toa ref="motion">motion</toa> at least **once**. When called, they do and must <toa ref="motion-vote">vote</toa> for that <toa ref="motion">motion</toa> at least **once**.</text>
+      <text id="💥">To call a <toa ref="motion-super">super motion</toa>, a <toa ref="constituent">CONstituent</toa> must have enough <toa ref="fungible-pc">PC</toa> to <toa ref="motion-vote">vote</toa> for that <toa ref="motion">motion</toa> at least **twice**. When called, they do and must <toa ref="motion-vote">vote</toa> for that <toa ref="motion">motion</toa> at least **twice**.</text>
     </section>
     <section id="🗳">
       <title>Voting on motions</title>
-      <text id="👽">All CONstituents have the inalienable right to vote on pending motions if they have enough pc to do so.</text>
-      <text id="📈">A CONstituent cannot change the "direction" (in favor or against) of their vote after their first vote, including 0-votes.</text>
-      <text id="💲">Each CONstituent's first vote on each motion costs 40 PC. The cost of each additional vote by that CONstituent on that motion is 40×(1.05^N) rounded down, where N is the number of votes that CONstituent has already cast on that motion.</text>
-      <text id="📰">Who voted on a motion, in which direction, when, and how many votes shall be a matter of public record.</text>
+      <text id="🗳">A <toa definition="true" ref="motion-vote">vote</toa> is an irreversible action taken by a <toa ref="constituent">CONstituent</toa> that costs <toa ref="fungible-pc">pc</toa> and potentially effects the result of <toa ref="motion">motions</toa>.</text>
+      <text id="👽">All <toa ref="constituent">CONstituents</toa> have the inalienable right to <toa ref="motion-vote">vote</toa> on <toa ref="motion-state-pending">pending</toa> <toa ref="motion">motions</toa> if they have enough <toa ref="fungible-pc">pc</toa> to do so.</text>
+      <text id="📈">A <toa ref="constituent">CONstituent</toa> cannot change the <toa definition="true" ref="motion-vote-direction">direction</toa> (in favor or against) of their <toa ref="motion-vote">vote</toa> after their first <toa ref="motion-vote">vote</toa>, including <toa ref="motion-zero-vote">0-votes</toa>.</text>
```

Comment:
motion-zero-vote is ref'd but not defined?

Reply:
Indeed, although I'd argue more an issue with the constitution than with this pull req…